spec/dummy/config/fashionista.conf

```diff
@@ -0,0 +1,7 @@
+base_url http://localhost:3000/
```

Comment:
I'm on it ![genie](https://cloud.githubusercontent.com/assets/19736358/26459716/b484f070-4177-11e7-9f82-b5e9eb6717d7.gif)